src/testrunner/testrunner.ts

```diff
@@ -713,7 +713,7 @@ export default class Testrunner extends EventEmitter {
         }
     }
 
-    private async _setFileInputDirect(selector: string, filePath: string, options?: { waitForVisible?: boolean, checkSelectorType?: boolean }) {
+    private async _setFileInputDirect(selector: string, filePath: string[], options?: { waitForVisible?: boolean, checkSelectorType?: boolean }) {
```

Comment:
Changing the signature is not enough, the [] wrapper around filePath have to be removed at line 742.